Tests/PackageModelTests/SwiftSDKTests.swift

```diff
@@ -554,4 +554,23 @@ final class DestinationTests: XCTestCase {
             parsedDestinationV2Musl
         )
     }
+
+    func testDefaultSDKs() throws {
+        let hostSDK = try SwiftSDK.hostSwiftSDK("/prefix/bin")
+
+        #if os(macOS)
+        let iOSTriple = try Triple("arm64-apple-ios")
+        let iOS = try XCTUnwrap(SwiftSDK.defaultSwiftSDK(for: iOSTriple, hostSDK: hostSDK))
```

Comment:
Does the SwiftPM test suite have a way to mock shell invocations? This invokes `xcrun` so it would be great to 1) verify that xcrun is indeed invoked and 2) mock the result.

Reply:
There's no such mocking available at the moment, unfortunately.

Reply:
and of course this is precisely the test that fails on CI 😶 Looks like it can't find iPhoneOS.sdk — could that be because the iOS component isn't installed in the runner's Xcode?

Reply:
Yes, that's quite likely, since we're not building for iOS on Swift CI. IMO better not to rely on that to be present.

Reply:
@MaxDesiatov do you know if this is still the case? Worth triggering CI again to confirm? If so I could add some plumbing to inject xcrun as a mockable dependency instead.